src/components/explore/AggregationsOverTimeGraph.svelte

```diff
@@ -213,36 +187,42 @@ $: if (referenceTextElement && referenceBackgroundElement) {
         x={hoverLabelPlacement}
         y={top} 
         width={hoverWidth}
-        height={height}
+        height={bottom - top}
```

Comment:
we can't store the result of this I suppose? Should be fine it only gets re-used 1 other time.